static/src/javascripts/bootstraps/enhanced/liveblog.js

```diff
@@ -39,8 +40,15 @@ const initFilterCheckbox = () => {
 }
 
 const createAutoUpdate = () => {
-    const isLiveblog = config.get('page.isLive')
-    updateBlocks({}, isLiveblog);
+    if (config.get('page.isLive')) {
+        updateBlocks();
+    }
+
+    bean.on(document.body, 'change', '.live-blog__filter-switch-label', () => {
```

Comment:
should we put this in it's own init function as its not related to autoUpdate anymore?